### PR TITLE
Fix app from not starting due to missing `solid-refresh` and bump version to `0.2.0`

### PR DESCRIPTION
**Pull request by brxken128**
On launch (at least on MacOS), the app will produce an error regarding Solid, Vite and `solid-refresh`. Navigating to `interface`, and running `pnpm i solid-refresh` fixes the issue.

I have tested this on a fresh clone, and it had the error, but this resolves the issue.

### CODE REVIEW

**Review by ameer2468 — DISMISSED** [review]

**Review by ameer2468 — APPROVED** [review]

File: apps/desktop/src-tauri/Cargo.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "sd-desktop"
-version = "0.1.4"
+version = "0.2.0"
```
Comment (oscartbeaumont) [inline apps/desktop/src-tauri/Cargo.toml:3]: Was this intentional?

Comment (oscartbeaumont) [reply apps/desktop/src-tauri/Cargo.toml:3]: Wait no you say it in the title. Just had read the description.

Comment (brxken128) [reply apps/desktop/src-tauri/Cargo.toml:3]: Yeah, we chose to just include the bump in here and to `0.2.0` as it was about to be merged anyway

Comment (Brendonovich) [reply apps/desktop/src-tauri/Cargo.toml:3]: sd-core also needs a version bump

**Review by Brendonovich — APPROVED** [review]

**Pull request merged**